client/packages/system/src/Item/api/hooks/useHistoricalStockLines/useHistoricalStockLines.tsx

```diff
@@ -12,7 +13,14 @@ export const useHistoricalStockLines = ({
   const sdk = getSdk(client);
   const { storeId } = useAuthContext();
 
-  const result = useQuery(`stocklinesAt${storeId}${itemId}${datetime}`, () =>
+  const key: string[] = [
```

Comment:
The type definition can be implied here, so don't really. need `: string[]`. Although if you're just adding it for clarity, then that's cool

Reply:
I think I added it when trying to resolve my confusion around the datetime, thought I had to convert it to string, but the realised it was just nullable
I think it's ok though?